### PR TITLE
Add schema:Role and related properties to codemeta.jsonld.

### PR DESCRIPTION
**Pull request by progval**
To have the properties suggested in #240 in Codemeta's context.

These are fairly specific to JSON-LD and probably don't have equivalents in other formats, should I add them to the crosswalk table anyway?

(I'm marking this PR as draft while we're discussing #240)

### CODE REVIEW

**Comment by moranegg** [comment]
This PR looks good!
if no more comments are added we will merge this PR at the end of March 2023 (included into v3.0)

Discussion will be open until March 15th and we will then proceed to a vote from March 15th to March 25th.

**Comment by moranegg** [comment]
Please add the added properties into the description file and into all .csv files.
Thanks!

**Comment by progval** [comment]
Done

File: properties_description.csv [hunk 1]
```diff
@@ -54,6 +54,9 @@ schema:Person,affiliation,Organization,"An organization that this person is affi
 schema:Person,identifier,URL,"URL identifier, ideally an ORCID ID for individuals, a FundRef ID for funders"
 schema:Person,name,Text,"The name of an Organization, or if separate given and family names cannot be resolved for a Person"
 schema:Person,address,PostalAddress or Text,Physical address of the item.
+schema:Role,endDate,Date or Datetime,The end date and time of the item, usually a Role. In ISO 8601 date format
+schema:Role,roleName,Text,URL,"A role played, performed, or filled by a person or organization. Recommended values: ""Design"" (algorithm), ""Architecture"", ""Debugging"", ""Maintenance"", ""Coding"", ""Documentation"", ""Testing"", ""Support"", ""Management""."
```
[the remark this answers is not in the document]
Comment (moranegg) [reply properties_description.csv:58]: Add etc. at the end

[the remark this answers is not in the document]
Comment (progval) [reply properties_description.csv:58]:
```suggestion
schema:Role,roleName,Text,URL,"A role played, performed, or filled by a person or organization. Suggested values: ""Design"" (algorithm), ""Architecture"", ""Debugging"", ""Maintenance"", ""Coding"", ""Documentation"", ""Testing"", ""Support"", ""Management""."
```

**Pull request merged**